# Split_The_Bill

First, I have a made a repository and inside of them I have created sln file, class library and test file.

And then, I have inserted three methods in class library and written the logic.

My methods has been build successfully.

In the next step, I have created first test case and i am getting passed the test. woohoo!!!

I  am not getting another test cases passed, I have tried many times and time was not there with me so I uploded how much I have completed.

I am getting the following error:Error Message:
   Assert.ThrowsException failed. No exception thrown. ArgumentException exception was expected. 
  Stack Trace:
     at Split_Test.UnitTest1.Amountsplited_ZeroAmount_ThrowsArgumentException() in /workspaces/Split_The_Bill/Split_Test/UnitTest1.cs:line 26
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)


